foundry.toml

```
@@ -0,0 +1,7 @@
+[profile.default]
+src = 'contracts'
+out = 'out'
+libs = ['node_modules', 'lib']
+test = 'test'
+cache_path  = 'cache_forge'
+no-match-contract = '.*EchidnaTest$'
```

I'd suggest enabling the optimization at compile time, with at least 10_000 runs.
```suggestion
no-match-contract = '.*EchidnaTest$'
optimizer = true
optimizer_runs = 10_000
```

If needed we can create a dev profile that doesn't run the optimizer and it's faster compiling:
```
[profile.development]
optimizer = false
```